content/post/data-warehouse-sla-p1.md

```diff
@@ -0,0 +1,64 @@
+
+In some ways, Analytics teams partially have themselves to blame - we have done such a great job espousing the benefits of data, but then we say you have to go through us to get it. As a result, the stakeholder relationship with the data warehouse is like a trust fall - the stakeholder needs to know the data will be there when they need it. If your data warehouse SLA is a promise, your consistent delivery on that promise will build tremendous amounts of trust.
+
+While the data warehouse SLA should articulate your commitment to deliver exemplary data services, it should also include commitments from the stakeholder side. After all, all good relationships require both parties to participate. The stakeholder-Analytics relationship requires commitment and collaboration from both sides.
+
+## Writing you SLA
+
+### The template
+The goal of writing an SLA to your internal stakeholders is not to create a detailed legal document that covers every possible situation. Instead, your SLA with your stakeholders is designed to get everyone on the same page. Issues with stakeholders are often a result of misalignment of expectations. In the spirit of formalizing the promise we are making to our stakeholders, with whom we already have a collaborative, friendly relationship, let's write... a letter. 
+
+Here is an SLA letter template partly inspired by the elements of an SLA described by Dinesh Verma in [Service Level Agreements on IP Networks](https://pdfs.semanticscholar.org/b526/8efaeac72624220bcdefa40cd44bf59aff90.pdf). Let's review the template here and then I will explore each section in more detail below and in part 2 of this post:
+
+>Dear [stakeholder],
+>
+>One of the core goals of the Data team is to empower you to make data driven decisions, both big and small. We are excited to see you using the data warehouse and business intelligence tool to run your area of the business, and we realize the exceptional amount of trust you have placed on us to be custodians of that data. As such, we would like to formalize our commitment to you. 
+>
+>The Data team promises to deliver **[services provided]** at **[expected performance]**. If there are any issues you can **[problem reporting]** and we promise to resolve the problem in **[response time]**. However, we have also established internal **[monitoring processes]** so that if there is an issue, we aim to find it first and **[issue communication]**.
+>
+>To help us continue to deliver these services, we ask for your **[stakeholder commitment]**
+>
+>Sincerely,
+>
+>The Data team
+
+
+### Stakeholder
+The first, and most important, element of the SLA is identifying stakeholder - who are you writing the letter to? You may have many stakeholders, each with different capabilities and use cases, which means different SLA requirements. Most of your stakeholder requirements will be slight variations (e.g., the same promise with different KPIs).
```

Comment:
Maybe in this section you could also talk about how to identify stakeholders that need a formal SLA vs those that don't?